duckduckgo/duckduckgo-privacy-extension
# Fix content script message exceptions for user allowlisted pages

The registeredContentScript and registeredTempAutofillContentScript
messages are sometimes still sent to the background, even for user
allowlisted websites. Take care to handle those messages correctly, to
avoid throwing exceptions (in background or content script) while
avoiding changing the behaviour of the content-scope and autofill
code.



**Reviewer:** @jonathanKingston 


## Steps to test this PR:
1. Install the extension.
2. Navigate to https://www.bristolpost.co.uk/ in a new tab and open the page console.
3. Click the extension icon and click to disable site protections. 
4. Open the extension background console and clear any messages.
5. Refresh the bristolpost tab.
6. Check that autofill, registeredContentScript and registeredTempAutofillContentScript exceptions are not shown in either console.

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
